File: app/src/main/java/com/BlizzardArmory/starcraft/Profile/Earned.java

package com.BlizzardArmory.starcraft.Profile;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Earned {

    @SerializedName("quantity")
    @Expose
    private int quantity;
    @SerializedName("startTime")
    @Expose
    private int startTime;

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public int getStartTime() {
        return startTime;
    }

    public void setStartTime(int startTime) {
        this.startTime = startTime;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/starcraft/profile/Earned.kt
package com.BlizzardArmory.model.starcraft.profile

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Earned.
 */
@Keep
data class Earned(

    @SerializedName("quantity")
    var quantity: Int,

    @SerializedName("startTime")
    var startTime: Int

)